docs/guide.mdx

```diff
@@ -0,0 +1,432 @@
+
+Below, we deploy a database using a transaction. We then call the blockchain using the returned transaction hash to verify that it was successful
+
+#### Deploy Database
+
+Database schema can be deployed on the Kwil nodes using `kwil-cli` as shown below. Refer to the [Kuneiform Guide](./category/kuneiform) to create your own database schema.
+:::tip
+You can write, compile, and deploy Kuneiform, as well as view sample Kuneiform files, online at https://ide.kwil.com.
+:::
+
+```bash title="Deploy a sample database"
+$ kwil-cli database deploy -p ./sample_test_db.kf --kwil-provider "localhost:50051"
+TxHash: 0b77fade7234277931cd43688ad1f3832623527e02993ce5c95946ab03774f92
+```
+
+#### Transaction status
+
+The transaction status can be queried using the transaction hash as follows:
+
+```bash
+$ kwil-cli utils query-tx 0b77fade7234277931cd43688ad1f3832623527e02993ce5c95946ab03774f92 --kwil-provider "localhost:50051"
+
+Transaction ID: 0b77fade7234277931cd43688ad1f3832623527e02993ce5c95946ab03774f92
+Status: success
+Height: 9912
+```
+
+## Join Kwil Network
+
+To add a new node to the existing Kwil network:
+1. Generate the required configuration as shown [above](./walkthrough#configuration).
+2. Obtain the `genesis.json` file for the desired network from another participant, and ensure that it is placed properly at the location `root_dir/abci/config/genesis.json`.
+3. Update the `chain.p2p.persistent_peers` with the peer addresses from the network using one of the methods described [here](./daemon/kwild#connecting-to-a-kwil-network) and start the node.
+
+  For example, the above node's p2p address is: `"b58257d0fd67c0757327ff16ec701990c24d5336@0.0.0.0:26656"`. 
+  ```toml 
+  [p2p]
+  "persistent_peers" = "b58257d0fd67c0757327ff16ec701990c24d5336@0.0.0.0:26656"
+  ```
+4. Start the node by running `kwild`.
+
+  ```bash
+  # Deploy Kwil Node
+  kwild --root_dir "path/to/root/dir"
+  ```
+
+5. Verify the connectivity to the existing node by running:
+  ```bash
+  # Check Peers connected to:
+  $ kwil-admin node peers
+  ```
+  ```json title="Connected Peers"
+  [
+      {
+          "node": {
+          "chain_id": "chain-Ba2NYd",
+          "name": "node-0",
+          "node_id": "b58257d0fd67c0757327ff16ec701990c24d5336",  // Peer ID
+          "proto_ver": 8,
+          "app_ver": 0,
+          "block_ver": 0,
+          "listen_addr": "tcp://127.0.0.1:26656",
+          "rpc_addr": "tcp://127.0.0.1:26657"
+          },
+          "inbound": false,
+          "remote_addr": "127.0.0.1"
+      }
+  ]
+  ```
+
+## Deploy a Multi-Node Kwil Network
+
+This section describes how to deploy a network with interconnected nodes from startup.
+
+Suppose you wish to deploy a network comprising three nodes, where two nodes act as validators, and one non-validator and the IP addresses of the nodes are `192.168.2.10`, `192.168.2.20` and `192.168.2.30` respectively.
```

Comment:
run-on sentence. Use something like: "and one non-validator.  The IP addresses of the nodes will be ..."